cpp/src/barretenberg/solidity_helpers/circuits/recursive_circuit.hpp

```diff
@@ -0,0 +1,144 @@
+#include "barretenberg/transcript/transcript.hpp"
+#include "barretenberg/plonk/proof_system/proving_key/serialize.hpp"
+#include "barretenberg/stdlib/primitives/curves/bn254.hpp"
+#include "barretenberg/stdlib/recursion/verifier/verifier.hpp"
+#include "barretenberg/stdlib/recursion/verifier/program_settings.hpp"
+#include "barretenberg/ecc/curves/bn254/fq12.hpp"
+#include "barretenberg/ecc/curves/bn254/pairing.hpp"
+
+//#include <transcript/transcript.hpp>
```

Comment:
removable

Reply:
Addressed in https://github.com/AztecProtocol/barretenberg/pull/363/commits/ca5f638df7a462a6b102d6a7e86a94759a95d335